client/components/data/document-head/index.jsx

```diff
@@ -16,13 +16,26 @@ import {
 	addDocumentHeadMeta as addMeta,
 	setDocumentHeadUnreadCount as setUnreadCount
 } from 'state/document-head/actions';
-import { getSelectedSiteId } from 'state/ui/selectors';
 
 class DocumentHead extends Component {
 	componentWillMount() {
-		this.props.setTitle( this.props.title );
-		this.props.setDescription( this.props.description );
-		this.props.setUnreadCount( this.props.unreadCount );
+		const {
+			title,
+			description,
+			unreadCount
+		} = this.props;
+
+		if ( title ) {
```

Comment:
> But I guess I can add checks for `null` vs `undefined`...

Actually, the other way round -> 044005f. There's now a helper function again, `getTitleAndSiteName`. I'm not exporting it, but it does live at `document-head/selectors` module scope. Personally, I find that cleaner than defining it inside `getFormattedTitle` (plus I'd need to rename some variables if I were to move it there to avoid shadowing). Are you okay with that, or would you suggest otherwise?